packages/apptive_grid_core/test/data_entity_test.dart

```diff
@@ -305,4 +305,34 @@ void main() {
       expect(entity.schemaValue, isNull);
     });
   });
+
+  group('Email', () {
+    test('Value is set', () {
+      final entity = EmailDataEntity('Value');
+
+      expect(entity.value, equals('Value'));
+      expect(entity.schemaValue, equals('Value'));
+    });
+
+    test('Default is null', () {
+      final entity = EmailDataEntity();
+
+      expect(entity.value, equals(null));
+    });
+  });
+
+  group('Phone Number', () {
+    test('Value is set', () {
+      final entity = PhoneNumberDataEntity('Value');
```

Comment:
For the sake of it please use a more sensible test value ;)

Reply:
done